titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/KubePodUtil.java

```diff
@@ -125,4 +128,13 @@
 
         return annotations;
     }
+
+    /**
+     * Returns a job descriptor with fields unnecessary for inclusion on the pod removed.
+     */
+    @VisibleForTesting
+    static  com.netflix.titus.api.jobmanager.model.job.JobDescriptor<?> filterPodJobDescriptor(com.netflix.titus.api.jobmanager.model.job.JobDescriptor<?> jobDescriptor) {
```

Comment:
Shouldn't this just be `filterPodJobDescriptor`? I don't see where a pod comes into the picture.